src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java

```diff
@@ -7,10 +7,11 @@
 @RequiredArgsConstructor
 public enum HearingChannel {
 
-    TELEPHONE("telephone", "Telephone", "Ffôn"),
-    VIDEO("video", "Video", "Fideo"),
-    FACE_TO_FACE("faceToFace", "Face To Face", "Wyneb yn wyneb"),
-    NOT_ATTENDING("notAttending", "Not Attending", null);
+    TEL("telephone", "Telephone", "Ffôn"),
```

Comment:
Is there a reason to have the enums named like this? From our perspective it's our code so we should call them something meaningful, unless I'm mistaken (like if you're calling getName() on them for example).

Reply:
Corrected